src/utils/stringFormater.ts

```diff
@@ -0,0 +1,10 @@
+export const snakeToCapitalization = (item: string) => {
```

Comment:
Let's rename file to string. It's already in utils folder.

Reply:
Sir these are common function so i have moved this to `Utils` file